src/main/rx/rx.c

```diff
@@ -153,7 +153,7 @@ void rxInit(rxConfig_t *rxConfig)
     }
 
     if (!feature(FEATURE_3D)) {
-        rcData[0] = rxConfig->rx_min_usec;
+        rcData[0] = rxConfig->rx_min_usec;  //wrong,why not rcData[3]???
```

Comment:
Yeah, if enable 3D feature, channel throttle's pwm is initialised midrc, 1500. so this place should reset rcData[3] to rx_min_usec. do you think so ?